pkg/util/helpers.go

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
These functions seems like they can be moved to a module as methods on the reconciler (or whatever) that uses them.

Reply:
I think these are best left in a general utility package since multiple our of controllers will need the capability to build restConfigs and controller-runtime clients. 

The Velero helpers will also need to be accessed by at least the MigStage and MigMigration controllers.